app/src/main/java/com/amaze/filemanager/adapters/data/LayoutElementParcelable.java

```diff
@@ -49,30 +52,34 @@
     public final String dateModification;
     public final boolean header;
 
+    @Nullable
```

Comment:
The field is Nullable but the constructor parameter is not? That's almost contradictory (and in Kotlin it is).

Reply:
Why do you need to save the context in the object? Can't you just save the date string?

Reply:
Oops, my bad. 04ba315f.